rspace/src/main/scala/coop/rchain/rspace/ReplayRSpace.scala

```diff
@@ -170,6 +170,7 @@ class ReplayRSpace[C, P, E, A, R, K](store: IStore[C, P, A, K], branch: Branch)(
               extractFirstMatch(channels, matchCandidates, channelToIndexedData) match {
                 case Right(None)             => runMatcher(comm, produceRef, remaining)
                 case Right(produceCandidate) => produceCandidate
+                case Left(_)                 => ???
```

Comment:
@goral09 Mateusz, can you advise how this should be handled?

Reply:
I don't know yet. We haven't yet figured out how to do cost accounting properly in the replay because we don't do work in RSpace during replay. At the moment, we never actually return `Left(OutOfPhloError)` because we start with `Integer.MAX_VALUE` as available phlos. I will fix that until Wednesday.